Digital-Umuganda/mbaza-health-backend
# Feat/ratings

### What does this PR do?

> - Add ratings so that linguists or health workers can rate conversations
> - Get statistics

### How can it be tested?

> - Pull and checkout to `feat/ratings` branch
> - Run the app and open `/docs` to see documentation
